pom.xml

```diff
@@ -151,6 +151,26 @@
                         </headerDefinitions>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>xml-maven-plugin</artifactId>
```

Comment:
I moved `xml-maven-plugin` next to `formatter-maven-plugin` at 347b67488dcc588d954a85b89d6ec34934b3182b commit .